doc/man3/PKCS5_PBKDF2_HMAC.pod

```diff
@@ -32,8 +32,7 @@ attempt to calculate the length of the B<salt> because it is not assumed to
 be NULL terminated.
 
 B<iter> is the iteration count and its value should be greater than or
-equal to 1. RFC 2898 suggests an iteration count of at least 1000. Any
-B<iter> less than 1 is treated as a single iteration.
+equal to 1. RFC 2898 suggests an iteration count of at least 1000.
```

Comment:
Please instead of removing the sentence replace it with corrected one. I.e., `... is treated as an error.`

Reply:
Sure